src/main/kotlin/com/quipalup/katydid/logentry/primaryadapter/httprest/LogEntryResource.kt

```diff
@@ -0,0 +1,10 @@
+package com.quipalup.katydid.logentry.primaryadapter.httprest
+
+import com.quipalup.katydid.common.domain.Id
+import com.quipalup.katydid.logentry.domain.LogEntry
+
+data class LogEntryResource(
+    val id: Id,
+    val type: String,
+    val attributes: LogEntry
+)
```

Comment:
```suggestion
data class LogEntryRequestResource(
    val type: String,
    val attributes: LogEntryResourceAttributes
)

data class LogEntryResponseResource(
    val id : String,
    val type: String,
    val attributes: LogEntryResourceAttributes
)

data class LogEntryResourceAttributes(
    val time: Long,
    val description: String,
    val amount: Int,
    val unit: String
)

data class LogEntryRequestDocument(
    val data: LogEntryRequestResource
)

data class LogEntryResponseDocument(
    val data: LogEntryResponseResource
)
```
This classes must have exactly the same data structure of request and response JSONs. You can compare this classes to the JSONs in the container test to verify by yourself how JSON data is mapped to data classes.